### PR TITLE
Completed prep-js-var exercise

### PR DESCRIPTION
**Pull request by annaaksenov**
prep-javascript-variables - Anna Aksenov
Screenshot (4)

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Anna Aksenov';
```
Comment (BrettAlbright) [inline prep-javascript-variables/index.html]: Hey Anna, just update this by changing `var` to `const` 

never should still be defined with `let`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Anna, just make sure to update the variable declarations to use const

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Anna, make sure to convert all of the variables except `never`

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**